### ISSUE TITLE
Appcenter aab uploading with fastlane plugin is wrong.

### ISSUE THREAD

**Issue by CristianMG** [microsoft/fastlane-plugin-appcenter#298]
Hi, I'm using the fastlane plugin to upload aab, the problem is that appcenter shows release as aab but when I click about download aab button option, it starting to download apk instead of aab.

![Captura de pantalla 2022-03-18 094921](https://user-images.githubusercontent.com/10176004/158967511-36796202-e6ac-4d7d-a5f3-2edbe503b156.png)
![Captura de pantalla 2022-03-18 094945](https://user-images.githubusercontent.com/10176004/158967519-e0a38607-44a8-4c5c-8ef4-564587d3a290.png)
![Captura de pantalla 2022-03-18 095008](https://user-images.githubusercontent.com/10176004/158967529-932401b8-52b2-49d8-aba9-e3a5e65c4c58.png)

The code related with this issue.

```
  desc "Building production bundle"
    lane :productionBundle do
      gradle(task: "app:bundle",flavor: "Production",build_type: "Release")
      distributeProductionBundle()
    end
```

```
  desc "Distribute production bundle"
     lane :distributeProductionBundle do
      uploadAppcenter(destination: "production",path: "#{lane_context[SharedValues::GRADLE_AAB_OUTPUT_PATH]}")
     end

```

When I upload aab from web It is ok uploaded as aab.

Thanks in advance.

Greetings.

**Comment by DmitriyKirakosyan** [comment]
Hi @CristianMG , thank you for you report!

I confirm that this is a bug. We will update this thread once the fix is released.

**Comment by CristianMG** [comment]
Hi, We still have the problem.

When we upload an aab, it fails because it is recognized as Apk, in fact, the type in the App Center is identified as "-". When we want to download aab is downloaded as an aab

Our code is something like this.


```
appcenter_upload(
  api_token: appcenter_api_key,
  owner_name: appcenter_user,
  owner_type: "user", # Default is user - set to organization for AppCenter organizations
  destinations: appcenter_group,
  app_name: appcenter_app_name,
  file: gradle_path , // #{lane_context[SharedValues::GRADLE_AAB_OUTPUT_PATH]}
  app_os: "Android",
  app_display_name: appcenter_app_name,
  notify_testers: true
)
```


Greetings

**Comment by rt-bishop** [comment]
Just wanted to add that when you click Download AAB and it starts downloading the APK file, the resulting file WILL actually be 
a correct AAB. You can see it for yourself if you try to download both and then analyze them via Android Studio. So as a temp fix all you need to do is just to change the extension name. Of course, it doesn't fix the issue, but until we got a proper fix it's better than nothing. Cheers!

**Comment by CristianMG** [comment]
@rt-bishop Thanks, It will be useful

**Comment by CristianMG** [comment]
I closed this issue by a missclick  Thanks, It will be useful to me, althought I'm still waiting for definitive solution @rt-bishop

**Comment by CristianMG** [comment]
Any news about this issue?

**Comment by aleksandr-dorofeev** [comment]
Hi @CristianMG
We are going to release a new version this week.

**Comment by AnatolyPristensky** [comment]
The fix is available in the latest release [v2.1.0](https://github.com/microsoft/fastlane-plugin-appcenter/releases/tag/v2.1.0)
Closing this issue.
